src/cr/cube/matrix/subtotals.py

```diff
@@ -593,3 +593,154 @@ def _subtotal_row(self, subtotal):
     def _table_margin(self):
         """Scalar np.float64 weighted-N for overall table."""
         return self._weighted_counts.table_margin
+
+
+class PairwiseSigTestSubtotals(_BaseSubtotals):
+    """Computes subtotal values for the pairwise t-test measure."""
+
+    def __init__(
+        self,
+        base_values,
+        dimensions,
+        column_proportions,
+        columns_base,
+        selected_column_idx,
+        diff_cols_nan=True,
+        diff_rows_nan=True,
+    ):
+        super(PairwiseSigTestSubtotals, self).__init__(base_values, dimensions)
+        self._column_proportions = column_proportions
+        self._columns_base = columns_base
```

Comment:
Can't you just pass `second_order_measures` here, instead of each measure separately? I'm talking about the 2 ones right above this comment. It'd be less messy if you could do that.

Reply:
good catch :D